helm/nginx-ingress-controller-app/values.yaml

```diff
@@ -236,38 +236,31 @@ controller:
   service:
 
     # controller.service.enabled
-    # If true, create NodePort Service. Dynamically calculated during cluster creation.
+    # If true, create LoadBalancer Service. Dynamically calculated during cluster creation.
     # Do not overwrite this value.
-    enabled: false
+    enabled: true
 
     # controller.service.externalTrafficPolicy
     # Configures kube-proxy, denotes if this Service desires to have external traffic routed to node-local or cluster-wide endpoints
     #   Local - kube-proxy only proxies requests to local endpoints (does not forward traffic to other nodes), source IP preserved
     #   Cluster - kube-proxy proxies requests randomly across all endpoints (forwards traffic to other nodes if necessary), source IP NAT'd
     externalTrafficPolicy: "Local"
 
-    # controller.service.type
-    # Applies only to clusters running on AWS.
-    # Valid values: external, internal
-    type: external
```

Comment:
I see `controller.service.type` config property name (with external/internal) to be technical debt, service type has meaning in k8s world, and I needed to use it for that purpose - in production we need nginx LoadBalancer Service but NodePort for integration test purposes only; tried with LB for integration tests but in kind LB never gets Ready

So I've:
- recycled `controller.service.type` for actual k8s Service type configuration
- replaced previous use of `controller.service.type` with a new `controller.service.public` flag

This breaks backward compatibility. I'm checking if it's actual issue.
If I'm not mistaken external/internal was feature available to aws node pools users only.